src/main/java/frc/robot/auto/Paths.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.auto;
+
+import java.util.HashMap;
+import java.util.Optional;
+
+import frc.robot.subsystems.Drive;
+
+public class Paths {
+  private static HashMap<String, Path> paths = new HashMap<>();
+  public static final Path NO_OP = new Path();
+
+  static {
+    paths.put("comp1", new Path().append(new GyroTurnSegment(180)));
```

Comment:
Is "comp1" displayed somewhere like on the dashboard? If so, may want to provide a more descriptive name.

Reply:
No, it isn't at least not yet.